### ISSUE TITLE
Planner Design Implementation

### ISSUE THREAD

**Issue by shivang-16** [Leadlly/leadlly.in#7]
**Issue:**  
The current landing page lacks a visual and interactive design for the Planner Plan feature, making it difficult for users to understand how it works. This issue aims to address that gap by implementing the Planner design.

**Description:**  
Implement the Planner design as demonstrated in the attached video. The goal is to create a responsive, user-friendly layout. Ensure that the design adheres to the visual and functional guidelines shown in the video.

UI Design and Resources (Web and Mobile):
All design resources are available in the Framer project: [Framer Project Link](https://framer.com/projects/Untitled--vJWJGFu70OOnsaZ47MmN-2RckG).
Explore the project to understand the design, and refer to the specific sections highlighted in the video. You can also use the code for components from Framer as a reference.

**Reference Video:**  

https://github.com/user-attachments/assets/c72c0c7d-e019-493b-bfbe-323d3f422548

In the video, you can see how the design is expected to work. By clicking the "Dive into Planner" button, a sliding window should open with a detailed explanation of the planner.

**Note:**  
The buttons for this feature are already present in the code (currently commented out with a different UI). Please use the existing buttons with that design only.

**Additional Context:**  
The attached video provides a detailed walkthrough of the Planner Plan feature. It should help guide you through the required design and interactions. For any questions or clarifications, feel free to leave a comment.

**Comment by vivek-bandi** [comment]
Hey I want to work on this , but where i can get those images?

**Comment by shivang-16** [comment]
is it not accessible throught framer link? ===> [Framer Project Link](https://framer.com/projects/Untitled--vJWJGFu70OOnsaZ47MmN-2RckG).

**Comment by vivek-bandi** [comment]
I am not able to export the images from framer


<img width="1280" alt="Screenshot 2024-10-08 at 9 19 06 AM" src="https://github.com/user-attachments/assets/deddcd9d-7926-42a3-8748-935714bacfa5">

**Comment by Rohit-Singh-Rawat** [comment]
Did u get them ?

**Comment by vivek-bandi** [comment]
Yeah , I saved the images from => [framer](https://sympathetic-tasks-219761.framer.app/)

**Comment by shivang-16** [comment]
hi @vivek-bandi are you working on this?

**Comment by vivek-bandi** [comment]
I tried but i'm not able to achieve this


<img width="1280" alt="Screenshot 2024-10-13 at 12 31 21 PM" src="https://github.com/user-attachments/assets/e34788a3-2e5c-4e95-bccb-73b124ccdddd">

**Comment by Kumar-s75** [comment]
Anyone working on it?

**Comment by shivang-16** [comment]
You can start if you want .

**Comment by Kumar-s75** [comment]
trying...
